bin/make_dependency_update_pr.py

```diff
@@ -51,17 +50,25 @@ def main():
         shell('git commit -a -m "Update dependencies"', check=True)
         run(
             [
-                'gh', 'pr', 'create',
-                '--repo', 'joerick/cibuildwheel',
-                '--base', 'master',
-                '--title', 'Update dependencies',
-                '--body', textwrap.dedent(f'''
+                'gh',
+                'pr',
+                'create',
+                '--repo',
+                'joerick/cibuildwheel',
+                '--base',
+                'master',
+                '--title',
+                'Update dependencies',
+                '--body',
+                textwrap.dedent(
+                    f'''
```

Comment:
Again, the left side makes more sense to me, having `'--repo', 'joerick/cibuildwheel'` on one line. And it's kind of hard to explain to black?

Reply:
`"--repo=joerick/cibuildwheel"`, done.